gridstatus/caiso/caiso.py

```diff
@@ -346,16 +347,23 @@ def _get_oasis(
         logger.info(f"Fetching URL: {url}")
 
         retry_num = 0
-        while retry_num < 3:
+        while retry_num < max_retries:
             r = requests.get(url)
 
             if r.status_code == 200:
                 break
 
             retry_num += 1
-            logger.error(f"Failed to get data from CAISO. Error: {r.status_code}")
-            logger.error(f"Retrying {retry_num}...")
```

Comment:
This doesn't need to be an error, and we definitely don't need to emit two errors every time we are rate limited. Sentry interprets error logs as errors which causes us to quickly exhaust our error budget.